pkg/controllers/status/cluster_status_controller.go

```diff
@@ -305,10 +305,12 @@ func (c *ClusterStatusController) initializeGenericInformerManagerForCluster(clu
 // and pod and start it. If the informer manager exist, return it.
 func (c *ClusterStatusController) buildInformerForCluster(clusterClient *util.ClusterClient) (typedmanager.SingleClusterInformerManager, error) {
 	singleClusterInformerManager := c.TypedInformerManager.GetSingleClusterManager(clusterClient.ClusterName)
-	if singleClusterInformerManager == nil {
-		singleClusterInformerManager = c.TypedInformerManager.ForCluster(clusterClient.ClusterName, clusterClient.KubeClient, 0)
+	if singleClusterInformerManager != nil {
+		return singleClusterInformerManager, nil
```

Comment:
I don't think we can just return here because we must ensure the informer for `Node and Pod` has been set up and started.

Reply:
Got it, it seems that the comments are somewhat distorted.

Reply:
Got it, it seems that my understanding has some errors.
/close